app/Http/Controllers/StatsController.php

```diff
@@ -187,7 +187,7 @@ public function downloaded(): \Illuminate\Contracts\View\Factory|\Illuminate\Vie
     public function seeders(): \Illuminate\Contracts\View\Factory|\Illuminate\View\View
     {
         // Fetch Top Seeders
-        $seeders = Peer::with('user')->select(DB::raw('user_id, count(*) as value'))->where('seeder', '=', 1)->groupBy('user_id')->latest('value')->take(100)->get();
+        $seeders = Peer::with('user')->select(DB::raw('user_id, count(distinct info_hash) as value'))->where('seeder', '=', 1)->groupBy('user_id')->latest('value')->take(100)->get();
```

Comment:
Would be better off using `count(distinct torrent_id)` instead of string `count(distinct info_hash)`